src/libostree/libostree-released.sym

```diff
@@ -416,6 +416,17 @@ global:
 LIBOSTREE_2017.9 {
 };
```

Comment:
I think I'd prefer to not touch it; I worry that it will show up as a red flag in [ABI checking programs](https://lvc.github.io/abi-compliance-checker/). etc.